components/SubmissionComments.tsx

```diff
@@ -1,19 +1,24 @@
-import React, { useContext } from 'react'
+import React, { useContext, useState } from 'react'
 import { Comment, Submission } from '../graphql'
 import Markdown from 'markdown-to-jsx'
 import ReviewerProfile from './ReviewerProfile'
 import { Button } from './theme/Button'
 import { TrashIcon } from '@primer/octicons-react'
 import styles from '../scss/submissionComments.module.scss'
 import { updateCache } from '../helpers/updateCache'
-import { useDeleteCommentMutation } from '../graphql'
+import { useDeleteCommentMutation, useEditCommentMutation } from '../graphql'
 import { GlobalContext } from '../helpers/globalContext'
+import { MdInput } from './MdInput'
+import _ from 'lodash'
 
 export const SubmissionComments: React.FC<{
   comments: Comment[]
   submission: Submission
 }> = ({ comments, submission }) => {
   const [deleteComment] = useDeleteCommentMutation()
+  const [editComment] = useEditCommentMutation()
+  const [editing, setEditing] = useState<undefined | number>()
```

Comment:
I think it's better to use `null | number` because undefined means this value is not assigned and we don't know it.

`undefined`: unintentional missing value
`null`: intentional missing value